client/search.html

```diff
@@ -0,0 +1,61 @@
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Google Font Icons -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="icon" href="./images/favicon.jpg">
+    <link rel="stylesheet" href="search-style.css" /> 
+    <!-- API -->
+    <title>Foolgle Search Results</title>
+</head>
+
+<body>
+    <div id="header">
+        <div id="topbar">
+            <img class="logo-img" src="./images/Foolgle.png" alt="Foolgle Logo Image">
+            <div class="search-bar">
+            <span class="material-icons">search</span>
+                <input id="search-input" class="search-input" type="text">
+            </div>
+            </div>
+        </div>
+        <div id="optionsbar">
+            <ul class="optionsmenu1">
+                <li class="optionsmenuactive">All</li>
+                    <li>News</li>
+                    <li>Videos</li>
+                    <li>Images</li>
+                    <li>Maps</li>
+                    <li>More</li>
+            </ul>
+        
+            <ul class="optionsmenu2">
+                    <li>Settings</li>
+                    <li>Tools</li>
+            </ul>
+        </div>
+    </div>
+    <div id="searchresultsarea">
+        <!--
+            CODE GOES HERE
```

Comment:
Does the search results go here or is there code missing from this document? This confused me lol